.github/workflows/push_pr.yml

```diff
@@ -8,87 +8,34 @@ on:
   pull_request:
 
 env:
-  TAG: "v0.0.0" # needed for goreleaser windows builds
-  REPO_FULL_NAME: ${{ github.event.repository.full_name }}
-  ORIGINAL_REPO_NAME: "newrelic/nri-jmx"
-  NRJMX_VERSION: '2.6.0'
-  DOCKER_LOGIN_AVAILABLE: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+  NRJMX_VERSION: '2.6.0' # TODO check if we should update it to latest release version
 
 jobs:
-  static-analysis:
-    name: Run all static analysis checks
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: newrelic/newrelic-infra-checkers@v1
-      - name: Semgrep
-        uses: returntocorp/semgrep-action@v1
-        with:
-          auditOn: push
-
-      - name: Install Go
-        uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
+  # reusable_push_pr contains static-analysis but it does not contain the Semgrep step
```

Comment:
If we managed to use the reusable static-analysis we we could get rid of the `tools` module.

Reply:
yep we can remove the commented static-analysis job as it is managed by reusable workflow.

Reply:
removed the commented static-analysis job